utils/texture-atlas.js

```diff
@@ -0,0 +1,119 @@
+import * as THREE from 'three';
```

Comment:
We already have many texture atlasing solutions; let's not add another one.

This should probably be an engine subsystem eventually, but if it's added to `utils` here then most likely it never will be.